src/XeroPHP/Models/Accounting/RepeatingInvoice/Schedule.php

```diff
@@ -193,18 +192,18 @@ public function setDueDate($value)
     }
 
     /**
-     * @return PaymentTerm
+     * @return string
      */
     public function getDueDateType()
     {
         return $this->_data['DueDateType'];
     }
 
     /**
-     * @param PaymentTerm $value
+     * @param string $value
      * @return Schedule
      */
-    public function setDueDateType(PaymentTerm $value)
+    public function setDueDateType(string $value)
```

Comment:
```suggestion
    public function setDueDateType($value)
```

I would remove this type-hint as the package supports 5.5+ and scalar type-hints are only available 7.0+.

I know that you are removing a type hint, which means nobody on a lower version of PHP is probably using this, but they also might be hitting an error and not reporting it. So maybe we remove this while making this change?

Then we can add type hints everywhere later down the line once we bump versions?

Reply:
Good call, done!